main.go

```
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"log"
+
+	"github.com/rayl17/academy-go-q42021/controller"
+	parameters "github.com/rayl17/academy-go-q42021/global"
+	"github.com/rayl17/academy-go-q42021/router"
+	service_csv "github.com/rayl17/academy-go-q42021/service/csv"
+	"github.com/rayl17/academy-go-q42021/usecase"
+)
+
+func main() {
+	println("Server is starting...")
+
+	pokemonService, _ := service_csv.NewPokemonService(parameters.CsvPath)
```

Never ignore errors.